php-practice/section11.php

```diff
@@ -252,15 +280,47 @@ function roll() {
  *
  * Create a Ball object and a BeachBall object.
  * Call roll() on both of them. Then echo out $color for both of them.
- * What happens when you call roll() on the Ball object?
- * What happens with the BeachBall object?
+ * What happens when you call roll() on the Ball object? 
+ * What happens with the BeachBall object? 
+ * ANSWER HERE: 'The ball is rolling'&'The beach ball is rolling, but it\'s slowing quickly'
  *
- * ANSWER HERE:
  */
 
+$ball = new Ball();
+
+echo $ball->roll();
+
+$beachBall = new BeachBall();
+
+echo $beachBall->roll();
```

Comment:
The roll function echoes something out. It doesn't return anything, so having `echo` at the beginning of this line doesn't really do anything.